app/storage/src/main/java/com/fsck/k9/preferences/migrations/StorageMigrationTo22.kt

```diff
@@ -0,0 +1,76 @@
+package com.fsck.k9.preferences.migrations
+
+import android.database.sqlite.SQLiteDatabase
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+
+/**
+ * Fix server settings by removing line breaks from username and password.
+ */
+class StorageMigrationTo22(
+    private val db: SQLiteDatabase,
+    private val migrationsHelper: StorageMigrationsHelper,
+) {
+    fun fixServerSettings() {
```

Comment:
`fixServerSettings` could be found 3 times in this class. I think they should have distinct names to improve readability.